vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -115,6 +124,9 @@ void setSelectAllCheckboxVisibility(
      * Returns whether the select all checkbox will be visible with the current
      * setting of
      * {@link #setSelectAllCheckboxVisibility(SelectAllCheckboxVisibility)}.
+     * <p>
+     * The select all checkbox will never be shown if the Grid uses lazy loading
+     * with unknown item count, i.e. no items count query provided to it.
```

Comment:
", meaning that no count callback has been provided."
IMO using "i.e." might is not maybe good

Reply:
Done